### PR TITLE
Add notifier configuration options

### PR DESCRIPTION
**Pull request by gwprice**
This change corresponds to notifier commits 73c67a9 and 88aa593.

@e0d @jimabramson

### CODE REVIEW

**Comment by jimabramson** [comment]
:+1:

File: playbooks/roles/notifier/defaults/main.yml [hunk 1]
```diff
@@ -14,6 +14,12 @@ NOTIFIER_LOG_LEVEL: "INFO"
 NOTIFIER_RSYSLOG_ENABLED: "yes"
 NOTIFIER_DIGEST_TASK_INTERVAL: "1440"
 
+NOTIFIER_DIGEST_EMAIL_SENDER: ""
```
Comment (e0d) [inline playbooks/roles/notifier/defaults/main.yml]: Aren't there sensible defaults for these?  Even if they cannot actually "work", say in the case of non-addressable email addresses, they have a didactic purpose for installers.

**Comment by e0d** [comment]
Other than my comment about defaults, I'm :+1:

File: playbooks/roles/notifier/defaults/main.yml (hunk 1 above)
Comment (gwprice) [inline playbooks/roles/notifier/defaults/main.yml]: Empty strings here are filtered out by the environment template, so the defaults in the notifier settings file will be used.

**Pull request merged**